pkg/dockerregistry/server/auth.go

```diff
@@ -415,17 +449,37 @@ func verifyPruneAccess(ctx context.Context, client client.SubjectAccessReviews)
 			Resource: "images",
 		},
 	}
-	response, err := client.SubjectAccessReviews().Create(&sar)
-	if err != nil {
-		context.GetLogger(ctx).Errorf("OpenShift client error: %s", err)
-		if kerrors.IsUnauthorized(err) || kerrors.IsForbidden(err) {
+	return withExponentialBackoff(func() error {
```

Comment:
sure... but if our strategy is to hang client connections and retry server-side, why isn't every single API call wrapped in a retry like this?